util/src/test/java/tech/pegasys/artemis/util/config/ArtemisConfigurationTest.java

```diff
@@ -74,4 +74,16 @@ void invalidMainnetArtemisConfig() {
     ArtemisConfiguration config = ArtemisConfiguration.fromString("deposit.numValidators=31");
     assertThrows(IllegalArgumentException.class, () -> config.validateConfig());
   }
+
+  @Test
+  void dataPathCanBeSet() {
+    final ArtemisConfiguration config = ArtemisConfiguration.fromString("output.dataPath=\".\"");
+    assertThat(config.getDataPath()).isEqualTo(".");
```

Comment:
should we test a different value to "."

Reply:
apparently its all changing, so i just copied the pattern... there'd be value if we were reading a file, but this test is a bit naff, i nearly didn't bother.